src/Modules/Modal.react.js

```diff
@@ -144,17 +183,17 @@ class Modal extends Component {
                                 style={containerInnerScrollStyles}
                             >
                                 {header ? React.Children.map(this.props.children, c => React.cloneElement(c, {
-                                    closeButton: closeButton,
-                                    inverse: inverse,
-                                    onClose: onClose,
-                                    title: title,
-                                    titleTruncate: titleTruncate
+                                    closeButton: this.props.closeButton,
+                                    inverse: this.props.inverse,
+                                    onClose: this._onClose,
+                                    title: this.props.title,
+                                    titleTruncate: this.props.titleTruncate,
```

Comment:
Same ... is there a reason we cannot use the destructured ones?

Reply:
Done.